mycore-user2/src/main/java/org/mycore/user2/login/MCRLDAPClient.java

```diff
@@ -91,160 +92,122 @@
  *
  * @author Frank Lützenkirchen
  */
+@SuppressWarnings({"PMD.ReplaceHashtableWithMap"})
+@MCRConfigurationProxy(proxyClass = MCRLDAPClient.Factory.class)
 public final class MCRLDAPClient {
```

Comment:
Is `final` suitable here?

Reply:
The class has no extension points for a subclass to modify, so I kept if final.